app/src/main/java/com/example/waterme/ui/WaterMeApp.kt

```diff
@@ -80,7 +81,7 @@ fun PlantListContent(
     LazyColumn(
         contentPadding = PaddingValues(dimensionResource(id = R.dimen.padding_medium)),
         verticalArrangement = Arrangement.spacedBy(dimensionResource(id = R.dimen.padding_medium)),
-        modifier = modifier
```

Comment:
To avoid overlapping, lets use safeDrawingPadding() instead.


<img width="623" alt="image" src="https://github.com/google-developer-training/basic-android-kotlin-compose-training-waterme/assets/24260141/c0b4c2f6-8832-4c04-b897-5350f803ddee">

Reply:
`.safeDrawingPadding()` causes an issue in portrait mode.

<img width="1097" alt="Screenshot 2023-08-25 at 9 46 30 AM" src="https://github.com/google-developer-training/basic-android-kotlin-compose-training-waterme/assets/7107817/b528cfa6-24f9-4cd4-88eb-14b0b6d464e2">

Reply:
Fixed.